app/views/books/index.html.haml

```diff
@@ -0,0 +1,11 @@
+.card-title.d-flex.justify-content-between
+  - if user_signed_in? && current_user.admin?
+    %h5.card-subtitle.mb-2.text-muted Welcome admin
+    = link_to "New book", new_book_path
+%h1.card-title Books
+= form_tag(search_path, :method => "get") do
```

Comment:
be careful when copy-pasting something
`:method => "get"`? it's not 2008 on the streets. `method: :get`